composer.json

```diff
@@ -13,9 +13,18 @@
         }
     ],
     "require": {
-        "symfony/symfony": "~2.3",
+        "php": ">=5.4.0",
+        "symfony/config": ">=2.3,<4",
+        "symfony/dependency-injection": ">=2.3,<4",
+        "symfony/event-dispatcher": ">=2.3,<4",
+        "symfony/http-kernel": ">=2.3,<4",
         "doctrine/doctrine-cache-bundle": "~1.0",
-        "php-tmdb/api": "~2.0"
+        "php-tmdb/api": "^2.0.14",
+        "twig/twig": "~1.11|~2.0"
+    },
+    "require-dev": {
+        "phpunit/phpunit": "^4.8",
+        "symfony/framework-bundle": ">=2.3,<4"
```

Comment:
Could you elaborate why the framework dependency is listed as a `dev` component?